builtin/logical/aws/path_roles.go

```diff
@@ -389,6 +406,7 @@ func (r *awsRoleEntry) toResponseData() map[string]interface{} {
 		"policy_arns":      r.PolicyArns,
 		"role_arns":        r.RoleArns,
 		"policy_document":  r.PolicyDocument,
+		"default_sts_ttl":  r.DefaultSTSTTL / time.Second,
```

Comment:
I have a slight preference for the `int64(r.DefaultSTSTTL.Seconds())` flow as that paradigm makes tests nicer -- when analyzing a returned map you get `int64` instead of a `time.Duration` that is actually an integer number of seconds even though time.Duration represents nanoseconds.

Reply:
Sure, don't have an opinion either way, was just pattern matching off other places in the code. Done.